app/cdap/components/Connections/Browser/GenericBrowser/index.tsx

```diff
@@ -139,21 +141,21 @@ export function GenericBrowser({ selectedConnection }) {
     );
   }
 
-  if (error) {
-    throw error;
-  }
-  if (!Array.isArray(entities) || (Array.isArray(entities) && !entities.length)) {
-    return <div>No entities available</div>;
-  }
+  const isEmpty = !Array.isArray(entities) || (Array.isArray(entities) && !entities.length);
+
   let headers = ['Name', 'Type'];
   const properties = objectQuery(entities, 0, 'properties') || [];
   headers = [...headers, ...properties.map((prop) => capitalize(prop.key))];
   const columnTemplate = headers.map(() => '1fr').join(' ');
   const getPath = (suffix) => (path === '/' ? `/${suffix}` : `${path}/${suffix}`);
 
-  const filteredEnitities = searchString.length
-    ? entities.filter((e) => e.name.includes(searchString))
-    : entities;
+  let filteredEntities = entities;
```

Comment:
Nit: this introduces a non-`const` binding, so I'd be inclined to write it without one. It works as is, of course:

```
const searchLower = searchString.toLocaleLowerCase();
const filteredEntities = searchLower.length
    ? entities.filter((e) => e.name.includes(searchString))
    : entities;
```

Reply:
Updated. I also added a `trim` to keep spaces from giving unexpected results.